{"uuid": "af1e21e0-76a5-4b65-a5e0-79fb9e3748b2", "children": ["5b2a937d-19cf-4ef3-ba98-41e18eb976a9"], "befores": [{"name": "pre_conditions_tc_001", "status": "broken", "statusDetails": {"message": "serial.serialutil.SerialException: could not open port 'COM7': FileNotFoundError(2, 'The system cannot find the file specified.', None, 2)\n", "trace": "  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Scripts\\Demo_Code\\test_001_demo_1.py\", line 19, in pre_conditions_tc_001\n    rly = Relay16CH('COM7')\n  File \"C:\\Users\\aldo.madrid\\Documents\\AutomationExample\\Hardware\\relay_usb.py\", line 6, in __init__\n    self._ser = serial.Serial(port=com_port, baudrate=baud_rate)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialwin32.py\", line 33, in __init__\n    super(Serial, self).__init__(*args, **kwargs)\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialutil.py\", line 244, in __init__\n    self.open()\n  File \"c:\\users\\aldo.madrid\\documents\\automationexample\\venv\\lib\\site-packages\\serial\\serialwin32.py\", line 64, in open\n    raise SerialException(\"could not open port {!r}: {!r}\".format(self.portstr, ctypes.WinError()))\n"}, "start": 1637780977189, "stop": 1637780978255}], "start": 1637780977189, "stop": 1637780978460}